### ISSUE TITLE
Add ability to list task queues for a given namespace

### ISSUE THREAD

**Issue by yiminc** [temporalio/temporal#3468]
Task queue in temporal is dynamic, they are created on demand. There is no pre-registration step needed. This is great because it is very easy to use and you can create dynamic task queues at runtime. 
There are cases where task queue is mis configured, or the generated task queue is incorrect. This could result in workflow been send to task queues that have no workers listening on. 

The ability to list all active task queues could be very useful to help diagnose these problems. Active task queues are the task queues that are loaded into matching service. Any task queues that have either new tasks added to or have poller waiting on them in last 5m is considered as active task queue.

This is not to be confused with all task queues that ever exists and been created in persistence. We don't want to go to persistence to load up all existing task queues. (Maybe we will in the future, but not for now.)

**Comment by jeacott1** [comment]
the ability to query for: task queue + registration(s) (with a count per type/impl of the existing state) would be great.
I'd like to be able to list all active workers too, and then show, node + process + worker + task queue + registration(s). 
In combination with a list any unmatched workflows/activities (that don't match any workers - is this possible already?), I think these could help in fault finding and tuning environments.

**Comment by jlegrone** [comment]
This seems to be a duplicate of https://github.com/temporalio/temporal/issues/1797
